packages/composites/bitcoin-json-rpc/src/endpoint/getblockchaininfo.ts

```diff
@@ -1,4 +1,4 @@
-import * as JSONRPC from '@chainlink/json-rpc-adapter'
+import * as JSONRPC from '@chainlink/json-rpc-adapter/src/adapter'
```

Comment:
Isn't the JSONRPC code available at the root from the compiled code in `/dist`?

Was this throwing an error?